CHANGES.md

```diff
@@ -32,6 +32,9 @@ Change Log
 * Added support for vertex shader uniforms when `tileset.colorBlendMode` is  `MIX` or `REPLACE`. [#5874](https://github.com/AnalyticalGraphicsInc/cesium/pull/5874)
 * Added separate file for the Cesium [Code of Conduct](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CODE_OF_CONDUCT.md). [#6129](https://github.com/AnalyticalGraphicsInc/cesium/pull/6129)
 * Fixed applying a translucent style to a point cloud tileset. [#6113](https://github.com/AnalyticalGraphicsInc/cesium/pull/6113)
+* Fixed sandcastle Particle System example for better visual
+* Fixed move forward and backward zooming for 2D camera.
+* Restricted access from calls to camera look functions while in 2D mode for sandcastle camera tutorial.
```

Comment:
Note the issue that you fixed like the other items. Also say what functions you update, and the change in functionality you made.